docs/core/docker/build-container.md

```diff
@@ -0,0 +1,391 @@
+---
+title: Containerize an app with Docker tutorial
+description: In this tutorial, you will learn how to containerize a .NET Core application with Docker.
+ms.date: 03/20/2019
+ms.topic: tutorial
+ms.custom: "mvc, seodec18"
+#Customer intent: As a developer, I want to containerize my .NET Core app so that I can deploy it to the cloud.
+---
+
+# Tutorial: Containerize a .NET Core app
+
+This tutorial teaches you how to build a Docker image that contains your .NET Core application. The image can be used to create containers for your local development environment, private cloud, or public cloud.
+
+In this tutorial you'll learn how to:
+
+> [!div class="checklist"]
+> * Create and publish a simple .NET Core app
+> * Create and configure a Dockerfile for .NET Core
+> * Build a Docker image
+> * Create and run a Docker container
+
+This tutorial teaches the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
```

Comment:
Again with "this tutorial" (and I know it's in the template. It's still repetitive.)

```suggestion
You'll understand the Docker container build and deploy tasks for a .NET Core application. The *Docker platform* uses the *Docker engine* to quickly build and package apps as *Docker images*. These images are written in the *Dockerfile* format to be deployed and run in a layered container.
```